tests/integration/ethernet_mtu_test.py

```diff
@@ -34,7 +34,6 @@ def test_increase_iface_mtu():
     eth1_desired_state = desired_state[INTERFACES][0]
     eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = 1900
-    eth1_desired_state['ipv6']['enabled'] = True
```

Comment:
Thre is a FIXME about IPv6 in this file that needs to be removed as well when these lines are removed.

Reply:
thanks, I missed that. Fixed.